README.md

```diff
@@ -67,9 +69,37 @@ Note: Common packages like `React`, `ReactDOM`, `Redux` and `ReactRedux` are not
 
 ## TaskRouter
 
-Before using this plugin you must first create a dedicated TaskRouter workflow or just add the following filter to your current workflow. Make sure it is part of your Flex Task Assignment workspace.
+Before using this plugin, you must first create a dedicated TaskRouter workflow or just add the following filter to your current workflow. Make sure it is part of your Flex Task Assignment workspace.
```

Comment:
@kaiquelupo Can they possibly add this to a separate workspace (i.e., not the "Flex Task Assignment")?

Reply:
No, they can't. When in a Flex project, you must use the Flex workspace.